lvgl-codegen/Cargo.toml

```diff
@@ -1,12 +1,23 @@
 [package]
 name = "lvgl-codegen"
 version = "0.1.0"
+description = "Code generation based on LVGL source code"
 authors = ["Rafael Caricio <rafael@caricio.com>"]
 edition = "2018"
+license = "MIT"
 publish = false
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+build = "build.rs"
 
 [dependencies]
-tera = "1.3.0"
 regex = "1.3.9"
+quote = "1.0.7"
+lazy_static = "1.4.0"
+clang = { path = "../../clang-rs" }
```

Comment:
Ok, I need to commit this to my own repo. The original `clang-rs` is not up to date with `clang-sys` that is used by `rust-bindgen`.

Reply:
If a specific commit of the `clang-rs` repo is needed, it's possible to do something like:
```text
clang = { git = "https://github.com/KyleMayes/clang-rs.git", rev = "<commit hash>" }
```

Reply:
I needed to actually write some tweaks to the code. 😕 Maybe a PR to upstream would be the way to go here.